client/src/components/ApplicationForm.tsx

```diff
@@ -0,0 +1,123 @@
+import { FormEvent, useContext, useState } from 'react';
+import { Job, submitApplication } from '../lib';
+import Button from './Button';
+import AppContext from './AppContext';
+
+type ApplicationFormProps = {
+  job: Job;
+};
+
+export default function ApplicationForm({ job }: ApplicationFormProps) {
+  const { user } = useContext(AppContext);
+  const [submitted, setSubmitted] = useState(false);
+
+  const inputClasses =
+    'rounded-lg p-2 bg-transparent border-[1.5px] border-stone-500 w-full mt-1';
+
+  async function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    try {
+      event.preventDefault();
+      const formData = new FormData(event.currentTarget);
+      await submitApplication(formData);
+      setSubmitted(true);
+    } catch (err) {
+      console.error(err);
```

Comment:
You want to display this error to the user somehow since they don't see the console.

For the final project you could just use `alert()`, though ideally it would be a nicer UI.